src/replication/protocol.rst

```diff
@@ -1585,20 +1585,12 @@ Replicator updates Replication Log on Source:
 
         PUT /source/_local/afa899a9e59589c3d4ce5668e3218aef HTTP/1.1
         Accept: application/json
-        Content-Length: 591
```

Comment:
I would prefer to explicitly show that this is not a chunked transfer. Removing content length will cause a questions.